packages/client/src/components/ProductReviews/ProductReviews.component.js

```diff
@@ -2,20 +2,31 @@ import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import { DisplayReviews } from './DisplayReviews/DisplayReviews.component';
 import './ProductReviews.styles.css';
+import ModalComponent from '../ModalComponent/ModalComponent';
 
 export const ProductReviews = ({ AllReviews }) => {
   const [reviews, setReviews] = useState([AllReviews[0]]);
+  const [openModal, setOpenModal] = useState(false);
 
   const handleMoreReviews = () => {
     if (reviews.length !== AllReviews.length) {
       const reviewsIndex = reviews.length;
       setReviews(reviews.concat(AllReviews[reviewsIndex]));
     }
   };
+  const handleModal = () => {
+    setOpenModal(false);
+  };
```

Comment:
```suggestion
  const toggleModal = () => {
    setShowModal(!showModal);
  };
```

This would be more generic.

Reply:
Done 👍